device-base/Dockerfile.armv7hf.rpi.tpl

```diff
@@ -5,7 +5,7 @@ LABEL io.resin.device-type="#{DEV_TYPE}"
 RUN echo "deb http://archive.raspbian.org/raspbian #{SUITE} main contrib non-free rpi firmware" >>  /etc/apt/sources.list \
 	&& apt-key adv --keyserver pgp.mit.edu  --recv-key 0x9165938D90FDDD2E \
 	&& apt-key adv --keyserver pgp.mit.edu --recv-keys 0x4404DDEE92BF1079 \
-	&& echo "deb http://resin-packages.s3-website-us-east-1.amazonaws.com/raspbian resin main" >> /etc/apt/sources.list
+	&& echo "deb http://resin-packages.s3-website-us-east-1.amazonaws.com/raspbian resin main" >> /etc/apt/sources.list.d/resin.list
```

Comment:
@petrosagg, I dont want to use COPY because  then we need to touch the generation script to copy that file in correct folder.